pkg/api/validation/dynakube/eec.go

```diff
@@ -3,11 +3,13 @@ package validation
 import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
 	"golang.org/x/net/context"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	errorExtensionExecutionControllerImageNotSpecified       = `DynaKube's specification enables the Prometheus feature, make sure you correctly specify the ExtensionExecutionController image.`
 	errorExtensionExecutionControllerInvalidPVCConfiguration = `DynaKube specifies a PVC for the extension controller while ephemeral volume is also enabled. These settings are mutually exclusive, please choose only one.`
+	warningIfmultiplyDKwithExtensionsEnabled                 = `You are already using a Dynakube that enables extensions. Having multiple Dynakubes with '.spec.extensions' enabled can have severe side-effects on “sum” and “count” metrics and cause double-billing.`
```

Comment:
```suggestion
	warningConflictingTenantForExtensions                 = `You are already using a Dynakube that enables extensions. Having multiple Dynakubes with '.spec.extensions' enabled can have severe side-effects on “sum” and “count” metrics and cause double-billing.`
```

Reply:
I like the message, but it doesn't mention that the problems is that both DynaKube use the same tenant (`apiUrl`), should we mention that for clarity? (if we mention it, then I would stick to calling out the `apiUrl` being the same, as maybe a customer will be not familiar with the "tenant" naming)

Reply:
@StefanHauth wdyt?

Reply:
renamed in bab9b2f5e

Reply:
Good catch, that should definitely be in the message. Also the name of the already existing DK should be in there.

```
`You are already using a Dynakube ('<name of existing DK>') that enables extensions. Having multiple Dynakubes with same '.spec.apiUrl' and '.spec.extensions' enabled can have severe side-effects on “sum” and “count” metrics and cause double-billing.`
```

Reply:
fixed in 40c4c4e5d531e55fe236236dad3b487658bffcfd

Reply:
@0sewa0, @StefanHauth  PTAL!